PP_APP/server/frontend/templates/ranking.html

```diff
@@ -1,13 +1,86 @@
 {% extends "base.html" %}
 {% load static %}
 {% block content %}
-
 <section id="about" class="about-section bg-light">
-    <div class="container">
-
-        <!-- PAGE CONTENT GOES IN HERE -->
-
-    </div>
+   <div class="container">
+      <h1 class="h3 mb-2 text-gray-800">Quick Stats</h1>
+      <!-- Content Row -->
+      <div class="row">
+         <!-- Time Chart -->
+         <div class="p-3">
+            <div class="card shadow">
+               <div class="card-body">
+                  <div class="chart-area">
+                     <canvas class="my-4 w-100 chartjs-render-monitor" id="myChart3" width="1043" height="200" style="display: block; height: 244px; width: 580px;"></canvas>
+                  </div>
+                  <hr>
+                  <p style="color:gray;"><small>Shows the amount of pointless packaging images submissions in this year.</small></p>
```

Comment:
"Shows the amount of pointless packaging images submissions in this year."

Should be changed to:

"The number of pointless packaging images submitted this year."